OrbitGl/App.cpp

```diff
@@ -176,15 +176,70 @@ void GLoadPdbAsync( const std::vector<std::wstring> & a_Modules )
     GModuleManager.LoadPdbAsync( a_Modules, [](){ GOrbitApp->OnPdbLoaded(); } );
 }
 
+//TODO: find a better name
+//-----------------------------------------------------------------------------
+void OrbitApp::StartRemoteCaptureBufferingThread()
+{
+    PRINT_FUNC;
+    m_MessageBufferThread = std::make_shared<std::thread>(&OrbitApp::ProcessBufferedCaptureData, this);
+    m_MessageBufferThread->detach();
```

Comment:
Using "detach" is a bit unusual - I would have expected that we join the thread especially since we keep m_Thread around.

Reply:
I switched to using join() in the stop method in the latest commit.
@pierricgimmig could you elaborate why you detached the threads in bpftrace.cpp?
We might want to use join instead of detach in BpfTrace.cpp, LinuxPerf.cpp and LinuxEventTracer.cpp.